### PR TITLE
Fixed reference URLs on delivery service (DNS, HTTP) pages

### PR DESCRIPTION
**Pull request by rimashah25**
## What does this PR (Pull Request) do?

- [x] This PR fixes reference URLs on Delivery Service Page, specifically for type: HTTP and DNS 

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?

1. Go to traffic_portal's create a delivery service page.
2. Select either HTTP, or DNS type
3. Go to section Cache Configuration Settings, and hover over every field in that section and click on the links present in help tool tip.
4. No link should show a `page does not exists yet` error.

## If this is a bug fix, what versions of Traffic Control are affected?

master (d809e9a)

## The following criteria are ALL met by this PR

- [x] I have explained why tests are unnecessary
- [x] I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR ensures that database migration sequence is correct OR this PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

## Additional Information

The PR doesn't need documentation or tests since it only updates the Reference URL path in Traffic Portal. No logic has been implemented.

### CODE REVIEW

File: traffic_portal/conf/config.js [hunk 1]
```diff
@@ -21,8 +21,8 @@
 module.exports = {
```
Comment (ocket8888) [inline traffic_portal/conf/config.js:21]: I don't think you meant to commit this file

Comment (rimashah25) [reply traffic_portal/conf/config.js:21]: oops..

**Review by ocket8888 — APPROVED** [review]
This looks like it works for what it changes - however, HTTP and DNS Delivery Service pages in TP still have two broken references. The first is in the "Deep Caching" field's tooltip, where it should probably be `https://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_router.html#deep-caching` to link to the correct anchor. The other is in the tooltip for "Geo Limit Redirect URL" and to be honest I'm not sure where that's supposed to take you.

Neither of these land on 404 pages, they're just non-existent anchors on an existent page. So you don't need to fix that if you don't want to, but I thought I'd bring it up.

**Comment by rimashah25** [comment]
> This looks like it works for what it changes - however, HTTP and DNS Delivery Service pages in TP still have two broken references. The first is in the "Deep Caching" field's tooltip, where it should probably be `https://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_router.html#deep-caching` to link to the correct anchor. The other is in the tooltip for "Geo Limit Redirect URL" and to be honest I'm not sure where that's supposed to take you.
> 
> Neither of these land on 404 pages, they're just non-existent anchors on an existent page. So you don't need to fix that if you don't want to, but I thought I'd bring it up.

I updated both these reference urls.

**Review by ocket8888 — APPROVED** [review]

**Pull request merged**